src/dialects/oracle/index.js

```diff
@@ -38,6 +38,8 @@ assign(Client_Oracle.prototype, {
     return require('oracle')
   },
 
+  SqlString: require('../../query/string'),
```

Comment:
This should not be needed since `src/client.js` base class already sets it to `src/query/string.js`